vaadin-checkbox-flow-parent/vaadin-checkbox-flow/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroup.java

```diff
@@ -84,6 +85,31 @@
  * <p>
  * Use CheckBoxGroup to group related items. Individual checkboxes should be
  * used for options that are not related to each other in any way.
+ * <h2>Validation</h2>
+ * <p>
+ * CheckboxGroup comes with a built-in validation mechanism that verifies that
+ * at least one checkbox is selected when
+ * {@link #setRequiredIndicatorVisible(boolean) required} is enabled.
+ * <p>
+ * Validation is triggered whenever the user initiates a value change by
+ * toggling a checkbox. Programmatic value changes trigger validation as well.
+ * If validation fails, the component is marked as invalid and an error message
+ * is displayed below the input.
```

Comment:
There is no single "input" in the group so maybe we could rephrase this:

```suggestion
 * is displayed below the group.
```

Reply:
Good catch, thanks, fixed.